universal-login-sdk/lib/api/DeployedWallet.ts

```diff
@@ -29,6 +32,13 @@ export class DeployedWallet implements ApplicationWallet {
   ) {
   }
 
+  private getContract(): Contract {
+    if (!this.contractInstance) {
+      this.contractInstance = new Contract(this.contractAddress, WalletContractInterface, this.sdk.provider);
```

Comment:
Why not just create it in constructor?

Reply:
Long story short - it would require tons of test refactoring, which could be done in future PRs.